cmd/crowdsec/crowdsec.go

```diff
@@ -119,9 +149,6 @@ func waitOnTomb() {
 			/*if it's acquisition dying it means that we were in "cat" mode.
 			while shutting down, we need to give time for all buckets to process in flight data*/
 			log.Warningf("Acquisition is finished, shutting down")
-			bucketCount := leaky.LeakyRoutineCount
-			rounds := 0
-			successiveStillRounds := 0
 			/*
 				While it might make sense to want to shut-down parser/buckets/etc. as soon as acquisition is finished,
```

Comment:
maybe kill the comm as it's not accurate anymore. Maybe add a quick explaination about the shutdown mecanism for future readers.